app/payments/heartland.php

```diff
@@ -288,74 +128,133 @@
     font-style: normal;
     font-weight: normal;
 }
+
+div.credit-card-submit.disable-button{
+	pointer-events: none;
+	opacity: 0.65;
+}
 </style>
 
 
 <div>
 
+    <script type="text/javascript">
+            GlobalPayments.configure({
+                "publicApiKey": "pkapi_cert_tTMCQaF3XoBGX9Afux"
```

Comment:
why are we hard-coding the key here?

Reply:
Mistakenly committed the working directory. Code is updated.
Please check